ta/pkcs11/src/processing_symm.c

```diff
@@ -713,3 +721,145 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 
 	return rc;
 }
+
+struct input_data_ref {
```

Comment:
structures that are inline of the code are hard to find -- could you either put it on header or at top of the file.

Reply:
My mistake. I will move it to the top of the file